### ISSUE TITLE
Send order email and PDF to a custom email address

### ISSUE THREAD

**Issue by iamtompickering** [verbb/gift-voucher#76]
**What are you trying to do?**
I would like to send an order email and the gift voucher PDF to a custom email address that is set in the order item options, rather than sending it to the order customer email address. 

I've had a look in the docs but can't see this as an option. 

Any idea how to get around this?

**Comment by engram-design** [comment]
I don't believe that would be a Gift Voucher function, instead a core Commerce function, as that's what controls the emails. Gift Voucher can attach the PDF to an existing Commerce email, but nothing more than that.

You can set the Recipient in Commerce to include Twig, so you could look up a line item's options?
![image](https://user-images.githubusercontent.com/1221575/139433882-48c33057-21b7-45b2-a496-c25b81fb27d2.png)

The code get might verbose, so I'd recommend using [preparse](https://plugins.craftcms.com/preparse-field) or similar to scope out the logic and just use that (with the preparse field stored on the order).

Let me know if I've misunderstood your request, but I don't believe its a Gift Voucher plugin issue.

**Comment by bymayo** [comment]
@engram-design Similar issue as what Tom is having is that we offer users to purchase a "digital" gift voucher. We've hacked around it by getting the recipients details (name and email) as well as the person sending it (again, name and email)

But ideally we'd like to send a reciept to the purchaser, and also send the gift voucher code via email to the recipient of the voucher.

I've managed to get around this by detecting when an order is complete, checking if the line item has a gift card in it and grabbing the `fromName` and `fromEmail` options.

Then passing this to a custom system message, which grabs the code and puts it all together in an email.

It works, but would ideally be good if this was a native feature of Gift Voucher plugin as it seems logical to do this if your buying a digital voucher.
